Meridian59/Data/Models/AdminData/AdminObject.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.ComponentModel;
+using Meridian59.Data.Lists;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    /// <summary>
+    /// Object that represents a displayed object in the Administration UI
+    /// </summary>
+    public class AdminObject : INotifyPropertyChanged
```

Comment:
I don't think this class correctly implements INotifyPropertyChanged.
Someone could change the vlaues for "ObjectNumber" or "ClassName" and they would not raise changed events.

To implement INotifyPropertychanged:
(1) Add constants for your property names to the beginning of the class
(2) Create protected fields, not only properties. Provide the fieldvalue in the getter of the property.
(3) Implement the setter of the property to check the value and if different, set the field and raise the propertychanged event.

Example for (3):
>> public uint NameRID
{
            get { return nameRID; }
            set
            {
                if (nameRID != value)
                {
                    nameRID = value;
                    RaisePropertyChanged(new PropertyChangedEventArgs(PROPNAME_NAMERID));
                }
            }
        }